custom/icds_reports/static/js/directives/location-filter/location-filter.directive.js

```diff
@@ -373,6 +373,21 @@ function LocationFilterController($rootScope, $scope, $location, $uibModal, loca
                     storageService.setKey('search', $location.search());
                     $scope.$emit('filtersChange');
                 }
+                if ($location.path().indexOf('poshan_progress_dashboard') !== -1 && selectedLocationIndex() > 0) {
+                    // if selected location is above state level when navigated to PPD, location is reset to state level
```

Comment:
can you explain this a bit more? If the location selected is above state level then it should load on National level no?

Reply:
Sorry I meant if the value of `selectedLocationLevel` is more (for district, block, sector and awc levels) then resetting it to state level